scripts/giphy.py

```diff
@@ -1,40 +1,37 @@
 """
 Respond to:
 
-    /giphy "phrase"
+    /giphy [tag]
 
-with a random image from giphy based on that phrase.
+with a random image from Giphy (optionally limited by tag)
 """
 
 import asyncio
 import functools
 import hangups
 import logging
-import re
 import requests
-import urllib
 
 
 logger = logging.getLogger('hangbot.scripts.giphy')
 
 
 API_KEY = 'dc6zaTOxFJmzC'  # Use the public API key for now
 API_URL = 'http://api.giphy.com/v1/gifs/random'
-COMMAND = '/giphy '
-MESSAGE_REGEX = re.compile(r'^{}'.format(COMMAND), re.IGNORECASE)
+COMMAND = '/giphy'
 
 
 async def process_message(client, conversation, message_event):
-    if MESSAGE_REGEX.match(message_event.text):
+    if message_event.text.startswith(COMMAND):
```

Comment:
Technically, that's not true. We could be processing a message that has no text, say if we wanted to do something with the attachments.